Makefile

```diff
@@ -48,17 +51,25 @@ ifndef HAS_PACKR2
 	go get -u github.com/gobuffalo/packr/v2/packr2
 endif
 
+HAS_DEP := $(shell command -v dep)
+dep:
+ifndef HAS_DEP
+	go get -u github.com/golang/dep/cmd/dep
```

Comment:
We should use a different command to install dep

```
curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

This gets the latest tagged release so that dep version works, and not the tip of master.

Reply:
For now, we've decided to remove fetching `dep` (This may change and/or be revisited in https://github.com/deislabs/porter/pull/201)